package.json

```diff
@@ -53,7 +53,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "postinstall": "husky install",
+    "prepare": "husky",
```

Comment:
While only installing dependencies, this can fail as husky wouldn't be installed. Are you sure we want that?

Reply:
afaik `prepare` runs after `install`. Tested with a fresh clone and everything seems to work.
This is the recommended approach from https://github.com/typicode/husky/releases/tag/v9.0.1